scripts/schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
what is this file used for? could we remove it, and have something print out the `schema` constant in config/schema.go? (or even better, have the structure deserialization do things?)

yes, maybe this suggestion is for a later PR - but having the same text twice doesn't feel like a good idea.